src/deploy/jenkins-uat.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('node-fetch');
+
+module.exports = () => {
```

Comment:
can we have a named function instead, so it is really clear what this is for, e.g.
```
// Inform Jenkins server to start an automation test run for the current branch.
function invokeAutomationTests() {
...
}
module.exports = invokeAutomationTests;